src/proto/robot_log_msg.proto

```diff
@@ -7,10 +7,11 @@ import "proto/tbots_timestamp_msg.proto";
 // the severity level of the log
 enum LogLevel
 {
-    DEBUG   = 0;
-    INFO    = 1;
-    WARNING = 2;
-    FATAL   = 3;
+    DEBUG    = 0;
+    INFO     = 1;
+    WARNING  = 2;
+    FATAL    = 3;
+    CONTRACT = 4;  // Special Fatal log for if a CHECK fails
```

Comment:
Curious why you called it contract?

Reply:
g3log names CHECK(...) the "Contract API" and their internal loglevel is also contract https://github.com/KjellKod/g3log/blob/7817fd3a9493a0f7577e1e5c63baa4b04953805a/src/g3log/loglevels.hpp#L132